cylc/flow/task_pool.py

```diff
@@ -378,29 +379,13 @@ def load_db_task_pool_for_restart(self, row_idx, row):
                 if timeout is not None:
                     itask.timeout = timeout
 
-            elif status in TASK_STATUSES_FAILURE:
-                itask.state.set_prerequisites_all_satisfied()
-
             elif status in (
                     TASK_STATUS_QUEUED,
                     TASK_STATUS_READY,
             ):
                 # reset to waiting as these had not been submitted yet.
+                # SOD: is this necessary?
```

Comment:
`is this necessary?` ...I'm curious? Is this to be decided at a later date?

Reply:
Good spotting. I had missed my own reminder comment here, should have prefixed it with "TODO"!

Reply:
Decided this is not necessary, and tested to confirm. I'll remove the code block and amend the method docstring a bit.